doc/website/getting-started/installation.md

```diff
@@ -79,6 +79,20 @@ X86_64:
 !!! attention
     Please ensure that the folder `/var/lock` exist and the filesystem supports file locking.
 
+### :fontawesome-brands-windows: Windows
+
+In case you do not have a Windows installation, Microsoft provides free developer images from [here](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/).
+
+Additionally, [CMake](https://cmake.org/download/) and [git](https://gitforwindows.org/) are required. The option to add CMake to the system PATH for all users should be set when it is installed.
+
+If the developer image from Microsoft is used, Visual Studio Community 2019 is already installed, else it can be found [here](https://visualstudio.microsoft.com/de/downloads/).
+
+To be able to compile iceoryx, the `Desktop development with C++` Workload must be installed. This is done by running `VisualStudioInstaller` and selecting the `Modify` button on `Visual Studio Community 2019`.
```

Comment:
Additionally you can install here git and cmake with Visual Studio selecting them in the `Individual Components` Tab. 
For cmake you can use `C++ CMake tools for Windows` and for git `Git for Windows` to install them.

Reply:
Is this usable outside of Visual Studio or just an integration into Visual Studio? Does it also apply to VS Code?